Source/bmalloc/libpas/src/libpas/pas_large_sharing_pool.c

```diff
@@ -531,11 +531,19 @@ static void splat_live_bytes(pas_large_sharing_node* node,
     bool did_overflow;
     switch (free_mode) {
     case pas_free:
+#if PAS_OS(WINDOWS)
```

Comment:
Why?

Reply:
`unsigned long` is 4 bytes on Windows and 8 bytes on Mac / Linux. This mismatch between `size_t` and `unsigned long` causes issues. I've replaced this with a cast from `size_t` to `unsigned long long` which should work on all the 64 bit platforms, but there may be issues if this is used on arm32.